src/Cornelis/Vim.hs

```diff
@@ -23,115 +23,89 @@ vimFirstLine = 0
 vimLastLine :: Int64
 vimLastLine = -1
 
-
-getWindowCursor :: Window -> Neovim env Pos
+getWindowCursor :: Window -> Neovim env AgdaPos
 getWindowCursor w = do
-  (row, col) <- window_get_cursor w
+  (oneIndexed -> row, zeroIndexed -> col) <- window_get_cursor w
```

Comment:
bug here? i think agda and vim index the lines the same way, but this function changes the indexing for lines to 0